src/ant/include/ant/AntennaChecker.hh

```diff
@@ -61,6 +64,62 @@ struct PARinfo;
 struct ARinfo;
 struct AntennaModel;
 
+///////////////////////////////////////
+// namespace gtl = boost::polygon;
+struct PinType;
+struct GraphNode;
+
+struct InfoType
```

Comment:
`InfoType` is too generic. What would you like to express here? The violation itself, the data computed for each gate?